urdf/body.urdf.xacro

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <!-- body defining macro -->
+  <xacro:macro name="body" params="
+    wheel_radius simulation_engine
+    imu_pos_x imu_pos_y imu_pos_z
+    imu_rot_r imu_rot_p imu_rot_y" >
+
+    <link name="base_link"></link> 
+
+    <joint name="base_link_to_body_link_joint" type="fixed">
+      <origin xyz="0.0 0.0 ${wheel_radius}" rpy="0.0 0.0 0.0" />
+      <parent link="base_link" />
+      <child link="body_link" />
+    </joint>
+
+    <link name="body_link">
+      <visual>
+        <geometry>
+          <mesh filename="file://$(find panther_description)/meshes/body.dae" />
+        </geometry>
+        <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
+      </visual>
+
+      <collision>
+        <geometry>
+          <mesh filename="file://$(find panther_description)/meshes/body_collision.stl" />
+        </geometry>
+        <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
+      </collision>
+
+      <inertial>
+        <origin xyz="-0.008 0.01 0.029" rpy="0.0 0.0 0.0"/>
+        <mass value="43.0" />
+        <inertia ixx="2.630036" ixy="-0.023480" ixz="0.052320"
+                                iyy="2.463509"  iyz="0.009337"
+                                                izz="4.611945" />
+      </inertial>
+    </link>
+
+    <joint name="body_link_to_imu_link_joint" type="fixed">
+      <origin xyz="${imu_pos_x} ${imu_pos_y} ${imu_pos_z}"
+              rpy="${imu_rot_r} ${imu_rot_p} ${imu_rot_y}" />
+      <parent link="body_link" />
+      <child link="imu_link" />
+    </joint>
+
+    <link name="imu_link"></link>
```

Comment:
I see that there is more of closing `</link>`, I prefer to close one liners with ` />`

Reply:
Done